add support for wx StockID menu items

RJVB opened the pull request:
Sets the new QAction's text and shortcut properties to the
values provided by the corresponding wxGetStock functions.

RJVB committed:
add support for wx StockID menu items

Sets the new QAction's text and shortcut properties to the
values provided by the corresponding wxGetStock functions.

Committed from host : Bola

vadz commented on src/qt/menu.cpp:
```diff
@@ -56,23 +57,54 @@ static void InsertMenuItemAction( const wxMenu *menu, const wxMenuItem *previous
+            {
+                itemAction->setText( wxQtConvertString( wxGetStockLabel( id ) ) );
+                wxAcceleratorEntry accel = wxGetStockAccelerator( id );
+                QString opcode(QStringLiteral("Ctrl+"));
```
This assumes that all accelerators use `Ctrl` and while this is indeed currently the case, I think it would be cleaner to still add `if ( accel.GetFlags() & wxACCEL_CTRL )` test here for the peace of mind.

vadz commented on src/qt/menu.cpp:
```diff
@@ -56,23 +57,54 @@ static void InsertMenuItemAction( const wxMenu *menu, const wxMenuItem *previous
+                }
+                else if ( accel.IsOk() )
+                {
+                    shortcut = wxQtConvertString( accel.ToRawString() );
```
Is this really correct? The string returned here for e.g. `wxID_COPY` should be "Ctrl+C", but `opcode` already contains "Ctrl+", won't this be a problem?

Or, IOW, why do we actually need `opcode` at all, couldn't we just use this string (and hard-coded "Ctrl+Q" for `wxID_EXIT`)?

vadz reviewed (commented):
Thanks for the patch!

We could merge it as is if it works in spite of duplicate "Ctrl+", but if it could be simplified a bit by dropping `opcode` and keeping just a single variable, it'd be even nicer.

RJVB commented:
>Is this really correct? The string returned here for e.g. `wxID_COPY` should be "Ctrl+C"

Ah, I clearly didn't look closely enough at what the code does, sorry about that. So you're saying that ToRawString() already contains the modifier keys in a sufficiently standard way. 

>, but `opcode` already contains "Ctrl+", won't this be a problem?

Nope, and it doesn't oblige you either to press both Ctrl keys ;)

>Or, IOW, why do we actually need `opcode` at all, couldn't we just use this string (and hard-coded "Ctrl+Q" for `wxID_EXIT`)?

Seems we should be able to, but let me check that first. When the CodeBlocks build against wxQt next fails... =)

RJVB committed:
simplify

wxAcceleratorEntry::ToRawString() is fully specified, no need to
add the modifiers opcode.

Committed from host : Bola

vadz commented:
Merged, thanks!
